internal/tiltfile/tiltfile_state.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/windmilleng/tilt/internal/model"
 )
 
+// TODO(dmiller) this needs to add all of the files in the directory as dependencies
```

Comment:
Should we have a `add_dir_as_dependencies` function?  Or is this a sign that we shouldn't be trying to implement this in Starlark, and should implement it in Go so that we can walk that chart directory and record dependencies?

Reply:
I think we need to have lots of thoughts about dependencies, and shouldn't block this change on that.

Are you sure that's the right way to get deps from helm? I don't know enough about it to know how else it could have deps. What does skaffold do?